client/blocks/login/index.jsx

```diff
@@ -129,7 +130,11 @@ class Login extends Component {
 	};
 
 	rebootAfterLogin = () => {
-		const { redirectTo } = this.props;
+		const { redirectTo, closeWindowAfterLogin } = this.props;
+
+		if ( closeWindowAfterLogin ) {
+			window.close();
```

Comment:
Login does get rendered on the server side, so unguarded statements like this leave me nervous. Can `rebootAfterLogin` ever run on the server? If so, I suggest changing this line to:

```suggestion
			typeof window !== 'undefined' && window.close();
```

Reply:
💯 Always guard the `window` 💂